test/test_helper.rb

```diff
@@ -102,15 +100,15 @@ class TestCase
     # The only drawback to using transactional fixtures is when you
     # actually need to test transactions.  Since your test is
     # bracketed by a transaction, any transactions started in your
-    # code will be automatically rolled back.
+    # code will be automatically rolled back. Defaults to true.
     self.use_transactional_tests = true
 
     # Instantiated fixtures are slow, but give you @david where
     # otherwise you would need people(:david).  If you don't want to
     # migrate your existing test cases which use the @david style and
     # don't mind the speed hit (each instantiated fixtures translates
     # to a database query per test method), then set this back to
-    # true.
+    # true. Defaults to false.
     self.use_instantiated_fixtures = false
     ##########################################################################
```

Comment:
`test_case.rb` is quite a long file and seems more complicated than it really is.

Do we still need these? They seem to date from the original setup of MO; the linked article is offline.
Both methods reiterate the defaults and don't seem to have changed in eons... but if we want to keep, i'm fine with it.